01-hello-git/README.md

```diff
@@ -66,194 +67,191 @@ or receiving changes from other repositories.
 
 ## Exercises
 
-### Creating a Fork
 
-A _fork_ is a personal copy of a repository hosted by a provider such as
-[GitHub](https://github.com). Create a fork and clone it.
+### Installing Git
+Before you start using Git, you have to make it available on your computer.
+click [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for 
+instulling on Linux, Mac and Windows.
+
+### Tell Git who you are
+First, you need to tell Git who you are:
 
 ```
-$ git clone git@github.com:jawnsy/training.git
-$ cd training
-$ ls
+$ git config --global user.email "you@example.com"
+$ git config --global user.name "Your Name"
+```
+
+### Give GitHub your public keys
+
+1- Paste the text Below, Substituting in your GitHub email address:
```

Comment:
Please consider installing the Markdown lint extension for VS Code: https://github.com/DavidAnson/vscode-markdownlint
This will help ensure your markdown files are formatted correctly